extensions/last-commit-info/src/main/java/de/sovity/edc/extension/LastCommitInfoService.java

```diff
@@ -0,0 +1,48 @@
+/*
+ *  Copyright (c) 2022 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+
+package de.sovity.edc.extension;
+
+import org.eclipse.edc.spi.system.ServiceExtensionContext;
+
+import java.util.Objects;
+import java.util.Scanner;
+
+public class LastCommitInfoService {
+
+    private final ServiceExtensionContext context;
+
+    public LastCommitInfoService(ServiceExtensionContext context) {
+        this.context = context;
+    }
+
+    public String getLastCommitInfo() {
+        var result = "Env Last Commit Info: \n";
+        result += getEnvLastCommitInfo() + "\n";
+        result += "Jar Last Commit Info: \n";
```

Comment:
should only be added if it differs from the env last commit info, and if it does, can be safely called "edc-extensions last commit info"